modules/tinymce/src/core/main/ts/table/TableFormats.ts

```diff
@@ -5,7 +5,8 @@
  * For commercial licenses see https://www.tiny.cloud/
```

Comment:
Should this file be in `core/fmt/` instead? As there's really no table logic in core anymore so having a `table` directory for this seems a bit odd. To be honest, this probably could even be merged into `DefaultFormats` 🤷

Reply:
Yeah, you are right. I have moved this file there. I would still like to keep it as a separate file for now. As part of this, I also moved a couple of tests files from `test/core/table` to `core/test/fmt` and `core/test/keyboard` instead